puppet/manifests/base.pp

```diff
@@ -8,9 +8,13 @@
   'npm': ensure => "present";
   'couchdb': ensure => "present";
   'libffi-dev': ensure => "present";
+  'libsqlite3-dev': ensure => "present";
   } ->
   class { 'ruby_install': 
     ruby_version => "2.2.1",
+  } ->
+  exec { 'mail-catcher':
+    command => 'gem install mailcatcher'
```

Comment:
Mail-catcher has dependencies that make it incompatible with a Gemfile install. It runs as a daemon, and requires a different build of event-machine than Rails does. I did attempt to bundle it with all the other gems and the Daemon broke.